scripts/main/contextMenu.js

```diff
@@ -591,6 +591,7 @@ contextMenu.shareAlbum = function (albumID, e) {
 				}
 			},
 		},
+		{ title: build.iconic("grid-two-up") + lychee.locale["QR_CODE"], fn: () => photo.qrCode(photoID) },
```

Comment:
```suggestion
		{ title: build.iconic("grid-two-up") + lychee.locale["QR_CODE"], fn: () => album.qrCode() },
```

I guess (not tested)?